.changes/1071-chat-screen-improvements.md

```diff
@@ -0,0 +1,2 @@
+- Hide emoji option after sent
+- Auto focus on chat input while swipe to reply
```

Comment:
You can take a look at the already included changes in this directory of how its written ;)